spring-cloud-dataflow-core/src/main/java/org/springframework/cloud/dataflow/core/dsl/DestinationNode.java

```diff
@@ -56,7 +56,7 @@ public String stringify(boolean includePositionalInfo) {
 
 	@Override
 	public String toString() {
-		return getDestinationName();
+		return ":" + getDestinationName();
```

Comment:
Good question Ilaya. The original approach Glenn was looking at was parsing the DSL - redacting the password in the argument value objects then converting that parsed AST back into DSL text. It became apparent we don't totally support that (never needed it before) - this was one place that needed changing but a more serious change is required in argument value processing because they lose whether they were quoted (and what quotes were used) when being parsed so converting back to DSL text isn't so easy. So we didn't do that extra work and instead Glenn went for a basic regex solution. This change to toString() remained. I don't mind it but strictly speaking it isn't necessary for the regex solution.